### ISSUE TITLE
Crawl the HTTPS version of the website first

### ISSUE THREAD

**Issue by mradkov** [aeternity/tipping-oracle-service#20]
Currently there is an UX problem in the claiming. 

Eg.
Tip sent to `https://example.com`
Manual claim: People try `www.example.com`, `http://example.com`, `https://example.com` which are all different.

So lets make the oracle crawl the HTTPS version of the website first and then fallback to non-secure one if nothing is found there.

**Comment by thepiwo** [comment]
I think this is a input issue, if the claim states `www.example.com`, this will be checked, default in web is port 80. The wallet could prefer https over http, the oracle service should stay neutral.

**Comment by kenodressel** [comment]
I agree here. Automating this could be abused if there is a different service running on the HTTPS port vs the HTTP port which, while not common, is totally possible.
